ios/PurchasesHybridCommon/PurchasesHybridCommonTests/Mocks/MockPurchases.swift

```diff
@@ -47,7 +47,7 @@ class MockPurchases: Purchases {
 
         let attributionPoster: AttributionPoster = AttributionPoster(
             deviceCache: deviceCache,
-            identityManager: identityManager,
+            currentUserProvider: identityManager,
```

Comment:
Maybe updated the `identityManager` name if that's no longer valid.

Reply:
good call, updating

Reply:
🤔 re-reading, the object itself _is_ an `IdentityManager`, instantiated a few lines above. It's just that `IdentityManager` conforms to `currentUserProvider`

Reply:
see https://github.com/RevenueCat/purchases-ios/pull/1462 for more details

Reply:
this change is very similar to what we have in https://github.com/RevenueCat/purchases-ios/pull/1462/files#diff-7551cde2d648c87ca17b22346479b25ab5e63186c40dcb3a411a1798ea8dd2d2